EventFilter/L1TRawToDigi/src/implementations_stage2/GTSetup.cc

```diff
@@ -70,17 +75,17 @@ namespace l1t {
                if (fed == 1404) {
```

Comment:
better if this magic number ended up in the FedRawData header file that has these numbers already